controllers.js

```diff
@@ -0,0 +1,14 @@
+const { selectCategories } = require("./models");
+
+exports.getCategories = (req, res, next) => {
+  console.log("in controllers");
```

Comment:
DOn't forget to remove logs before submitting a PR this code should be ready to merge to production in theory 👍